zulip_botserver/zulip_botserver/server.py

```diff
@@ -56,7 +62,11 @@ def handle_bot(bot):
     except SystemExit:
         return BadRequest("Cannot fetch user profile for bot {}, make sure you have set up the flaskbotrc "
                           "file correctly.".format(bot))
-    message_handler = bots_lib_module[bot].handler_class()
+
+    lib_module = get_bot_lib_module(bot)
+    if lib_module is None:
+        return BadRequest("Cannot fetch Bot Handler Class for bot {}.".format(bot))
```

Comment:
It looks like bots that weren't in `zuliprc` will fail the `available_bots` check above, but bots that had a misconfigured `zuliprc` will fail this check.  However, I don't think this distinction is necessary; for the sake of simplicity, would it make sense to remove the `available_bots` check and move this `lib_module` stuff to the top of the function?